adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -184,6 +184,58 @@ def detect_taps(self, value):
                     value, 60, time_limit=10, time_latency=50, time_window=255
                 )
 
+    # pylint: disable-msg=too-many-arguments
+    # Many default arguments which will probably not need changing in most cases
+    def configure_taps(
+        self,
+        value,
+        accel_range=adafruit_lis3dh.RANGE_8_G,
+        theshold=None,
```

Comment:
`theshold` is misspelled - it should be `threshold`. This will need to be updated throughout the rest of the function as well.

Reply:
Fixed.